src/universal/components/RetroSidebarDiscussSection.js

```diff
@@ -67,16 +43,29 @@ const RetroSidebarDiscussSection = (props: Props) => {
         const {reflectionGroup} = stage;
         if (!reflectionGroup) return null;
         const {title, voteCount} = reflectionGroup;
+        const isOutOfSync = false;
```

Comment:
@mattkrick similar to `AgendaItem.js` this is where I need to know state of nav for active, complete, and out of sync … I’m guessing we may be able to add to the query?